docs/docs/hooks.md

```diff
@@ -21,15 +21,19 @@ For a full list of hooks, see [Hooks API](/api/hooks).
   beforeCreate(instance, options, fn)
   beforeDestroy(instance, options, fn)
   beforeUpdate(instance, options, fn)
+(5)
+  beforeSave(instance, options, fn)
```

Comment:
It should run with whatever logic hook normally does.
Which should happen automatically if we simply append the hook to the proxy targets when a hook with proxy targets is defined.